tests/test_splitter.py

```diff
@@ -106,3 +168,138 @@ def test_switchback_balanced_splitter_abc():
         counts = sampled_treatment.treatment.value_counts()
         assert counts.max() == 2
         assert counts.min() == 1
+
+
+def assert_balanced_strata(
+    df, strata_cols: List[str], cluster_cols: List[str], treatments: List[str]
+):
+
+    splitter = StratifiedClusteredSplitter(
+        strata_cols=strata_cols, cluster_cols=cluster_cols
+    )
+
+    treatment_df = splitter.assign_treatment_df(df)
+
+    treatment_df_unique = treatment_df[
+        strata_cols + cluster_cols + ["treatment"]
+    ].drop_duplicates()
+
+    ##
+    if len(cluster_cols) > 1:
```

Comment:
can we extract this into a function? the whole if: agg else

Reply:
done! `combine_columns(df, cols_list)`